### ISSUE TITLE
Precheck complains about "todo" in Spannish and Portuguese

### ISSUE THREAD

**Issue by Johennes** [fastlane/fastlane#9567]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description

The precheck barks when "todo" appears as a substring.

```
[15:56:38]: Checking app for precheck rule violations
[15:58:02]: ✅  Passed: No negative  sentiment
[15:58:02]: 😵  Failed: No placeholder text-> using placeholder text (e.g.:"lorem ipsum", "text here", etc...)
[15:58:02]: 😵  Failed: No mentioning  competitors-> mentioning other platforms, like Android or Blackberry
[15:58:02]: ✅  Passed: No future functionality promises
[15:58:02]: ✅  Passed: No words indicating test content
[15:58:02]: ✅  Passed: No curse words
[15:58:02]: ✅  Passed: Incorrect, or missing copyright date
[15:58:41]: ✅  Passed: No broken urls
+--------------------------------+----------------+
|               Potential problems                |
+--------------------------------+----------------+
| Field                          | Failure reason |
+--------------------------------+----------------+
| description: (pt-PT)           | found: todo    |
| description: (es-ES)           | found: todo    |
| description: (es-MX)           | found: todo    |
| description: (pt-BR)           | found: todo    |
```

However, this can easily happen in some languages and doesn't indicate a placeholder. Example:

```• Organiza todos tus planes de viaje en un solo lugar sin importar donde reserves.```

The rule should probably only trigger when "todo" is the only string in the file.

### Environment

Please run `fastlane env` and copy the output below. This will help us help you :+1:
If you used `--capture_output` option please remove this block - as it is already included there.

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.12.4                                     |
| Ruby                        | 2.4.1                                       |
| Bundler?                    | false                                       |
| Git                         | git version 2.11.0 (Apple Git-81)           |
| Installation Source         | /usr/local/bin/fastlane                     |
| Host                        | Mac OS X 10.12.4 (16E195)                   |
| Ruby Lib Dir                | /usr/local/Cellar/ruby/2.4.1_1/lib          |
| OpenSSL Version             | OpenSSL 1.0.2k  26 Jan 2017                 |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 8.3                                         |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE |             |   |

### fastlane files:

**No Fastfile found**

**No Appfile found**

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.41.0  | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.1.2        |
| slack-notifier            | 1.5.1        |
| CFPropertyList            | 2.3.5        |
| claide                    | 1.0.1        |
| colored2                  | 3.1.2        |
| nanaimo                   | 0.2.3        |
| xcodeproj                 | 1.4.4        |
| public_suffix             | 2.0.5        |
| addressable               | 2.5.1        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| tty-screen                | 0.5.0        |
| babosa                    | 1.0.2        |
| colored                   | 1.2          |
| highline                  | 1.7.8        |
| commander-fastlane        | 4.4.4        |
| excon                     | 0.55.0       |
| faraday                   | 0.12.1       |
| unf_ext                   | 0.0.7.4      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20170404 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| fastimage                 | 2.1.0        |
| gh_inspector              | 1.0.3        |
| declarative               | 0.0.9        |
| declarative-option        | 0.1.0        |
| representable             | 3.0.4        |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| little-plugger            | 1.1.4        |
| multi_json                | 1.12.1       |
| logging                   | 2.2.2        |
| jwt                       | 1.5.6        |
| memoist                   | 0.15.0       |
| os                        | 0.9.6        |
| signet                    | 0.7.3        |
| googleauth                | 0.5.1        |
| httpclient                | 2.8.3        |
| google-api-client         | 0.12.0       |
| mini_magick               | 4.5.1        |
| multi_xml                 | 0.6.0        |
| rubyzip                   | 1.2.1        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 0.0.4        |
| faraday_middleware        | 0.11.0.1     |
| json                      | 2.1.0        |
| io-console                | 0.4.6        |
| openssl                   | 2.0.3        |
| plist                     | 3.3.0        |
| bundler                   | 1.15.0       |
| terminal-table            | 1.8.0        |
| unicode-display_width     | 1.2.1        |
</details>

*generated on:* **2017-06-23**
</details>

**Comment by evilDave** [comment]
Just noticed this too. +1.

**Comment by glancashire** [comment]
Also complains about "Notizen" in german, as it looks for "tizen"....

**Comment by mpirri** [comment]
This issue was fixed in _fastlane_ 2.40.0.  I'm going to close this issue now, please feel free to reopen this issue if you are still having difficulty.

Thanks so much for contacting us about this!  🚀
